resources/ext/config/conf.d/puppetserver.conf

```diff
@@ -1,14 +1,20 @@
+# Where the puppet-agent dependency places puppet, facter, etc...
+# Puppet server expects to load Puppet from this location
+os-settings: {
+    ruby-load-path: [/opt/puppetlabs/puppet/lib/ruby/vendor_ruby]
+}
+
 # configuration for the JRuby interpreters
 jruby-puppet: {
     # This setting determines where JRuby will look for gems.  It is also
     # used by the `puppetserver gem` command line tool.
-    gem-home: /var/lib/puppet/jruby-gems
+    gem-home: /opt/puppetlabs/server/data/puppetserver/jruby-gems
 
     # (optional) path to puppet conf dir; if not specified, will use the puppet default
-    master-conf-dir: /etc/puppetlabs/agent
+    master-conf-dir: /etc/puppetlabs/puppet
 
     # (optional) path to puppet var dir; if not specified, will use the puppet default
-    master-var-dir: /opt/puppetlabs/agent/cache
+    master-var-dir: /opt/puppetlabs/server/data/puppetserver
```

Comment:
I generally agree with your comments, @joshcooper, but that postinst step will only guarantee that `puppet master --configprint vardir` will work *if* the files aren't modified later.  So it should be fine for beaker/acceptance tests unless there are tests that manipulate the vardir setting, just something for us to be aware of.

And yes, I'm thinking that in P5 we should remove the 'master' command altogether and add tools for interrogating the necessary settings from Puppet Server directly.  I don't think we have any intention of adding any new beaker settings for that, totally agree with your statement.